ios/Runner.xcodeproj/xcshareddata/xcschemes/Runner.xcscheme

```diff
@@ -15,7 +15,7 @@
             <BuildableReference
                BuildableIdentifier = "primary"
                BlueprintIdentifier = "97C146ED1CF9000F007C117D"
-               BuildableName = "SEEDS Wallet.app"
+               BuildableName = "Runner.app"
```

Comment:
Is it necessary?

Reply:
These changes happen automatically when I run the app on my iOS device. Should I revert these changes?